tests/acceptance/products-test.js

```diff
@@ -20,47 +20,40 @@ describe('Acceptance | products', function() {
     server.create('product', { name: 'Personal Computer' });
     await page.visit();
 
-    // show 
+    // show
     await page.products(0).open();
     expect(page.productName).to.include('Personal Computer');
 
     // edit
-    await page.openProducts().edit()
-      .editProductName('Apple 1')
-      .saveProduct();
+    await page.openProducts().edit().editProductName('Apple 1').saveProduct();
```

Comment:
would be nice to change prettier in the way that each `.functionCall` would be in new line ...